Docs/source/install/hpc/lxplus.rst

```diff
@@ -0,0 +1,141 @@
+.. _building-lxplus:
+
+LXPLUS (CERN)
+=============
+
+The LXPLUS cluster is located at CERN.
+
+* `Lxplus documentation <https://lxplusdoc.web.cern.ch>`__
+* Batch system: `HTCondor <https://batchdocs.web.cern.ch/index.html>`__
+* Filesystem locations:
+    * User folder: `/afs/cern.ch/user/<a>/<account>` (10GByte)
+    * Work folder: `/afs/cern.ch/work/<a>/<account>` (100GByte)
+    * Eos storage: `/eos/home-<a>/<account>` (1T)
+
+Through LXPLUS we have access to CPU and GPU nodes (the latter equipped with NVIDIA A100 or T4 GPUs).
+
+Installation
+------------
+Only very little software is pre-installed on Lxplus so we show how to install from scratch all the dependencies using Spack.
+
+For size reasons it is not advisable to install WarpX in the $HOME directory, while it should be installed in the "work directory". For this purpose we set an environment variable with the path to the "work directory"
+
+.. code-block:: bash
+
+    export WORK=/afs/cern.ch/work/<a>/<account>/
+
+GCC
+^^^
+The pre-installed GNU compiler is outdated so we need a more recent compiler. Here we use the gcc 9.2.0 from the LCG project, but other options are possible.
+
+We activate it by doing
+
+.. code-block:: bash
+
+    source /cvmfs/sft.cern.ch/lcg/releases/gcc/9.2.0/x86_64-centos7/setup.sh
+
+In order to avoid using different compilers this line could be added directly into the ``$HOME/.bashrc`` file.
+
+Spack
+^^^^^
+We download and activate Spack in ``$WORK``:
+
+.. code-block:: bash
+
+    cd $WORK
+    git clone -c feature.manyFiles=true https://github.com/spack/spack.git
+    source spack/share/spack/setup-env.sh
+
+When installing packages Spack will try to set permissions in a way which is forbidden on the LXPLUS file system (AFS), but we can avoid this by modifying the ``spack/etc/spack/defaults/config.yaml``. Inside this file we need to change the line
+
+.. code-block:: bash
+
+    allow_sgid: true
+
+to
+
+.. code-block:: bash
+
+    allow_sgid: false
```

Comment:
Easier to run a command than changing a file (and the files in `spack/etc/spack/defaults/` should not be changed usually, but their counter-parts in `$HOME/.spack/` are):
```suggestion
When installing packages Spack will try to set permissions in a way which is forbidden on the LXPLUS file system (AFS), but we can avoid this setting the Spack option:

.. code-block:: bash

    spack config add "config:allow_sgid:false"
```

Reply:
Oh I didn't know this one. Thanks a lot 😄